lib/unleash/client.rb

```diff
@@ -117,7 +129,7 @@ def info
         'appName': Unleash.configuration.app_name,
         'instanceId': Unleash.configuration.instance_id,
         'sdkVersion': "unleash-client-ruby:" + Unleash::VERSION,
-        'strategies': Unleash.strategies.keys,
+        'strategies': nil,
```

Comment:
What do we want to do here? Accessing strategies isn't really sane anymore but we can yield the names of the strategies themselves. Not actually sure what the purpose of exposing this is right now

Reply:
Okay, I've changed this, this now yields a hash of name: {strategy_name}, which is what the Unleash API is expecting - this is needed for registration